VMEncryption/main/EncryptionSettingsUtil.py

```diff
@@ -30,6 +30,8 @@
     import http.client as httpclient #python3+
 except ImportError:
     import httplib as httpclient #python2
+
+import xml.etree.ElementTree as ET
```

Comment:
Do we need to separately install this package or we expect it to always be there?

Reply:
We expect this package to be there. It is part of python standard library